### DISCUSSION TITLE
Duda sobre el curso.

### DISCUSSION THREAD

**Discussion by GuillenLuis** [Launch-X-Latam/MisionFrontEnd#457] [Team Posts]
Hola, compañeros y compñeras, espero estén teniendo un excelente día, tarde o noche. Disculpen, tengo una duda: ando muy atrasado con la parte de Frontend; aún estoy por terminar la práctica 1. Me está tomando mucho más tiempo del que esperaba, mas que soy algo lento para hacer las cosas. Mi pregunta es si hay alguna fecha límite para terminar las práticas y lo que es la Misión Frontend para pasar a la del Backend o si pasa algo si no llegamos a terminar todos los temas dentro de una fecha en concreto. Me preocupa que me llegue a afectar en algo. Estoy tratando de ponerme al corriente lo más rápido posible, pero por el tiempo, me está costando. 

Gracias y un saludo. :')

**Comment by gumodi35** [comment]
Que tal, no te preocupes hay 🕐  tiempo, si no me equivoco es hasta el 4 de Noviembre la fecha para entregar, así que creo que puedes 🧑‍🏭 hacerlo. <br>
Si hubiera muchos que faltan puede que el **mission Comander** de más tiempo! Cualquier duda manda DM. Saludos! 👍

**Comment by GuillenLuis** [comment]
Hola. Muchas gracias. 🥺

**Comment by lucia123lu** [comment]
Yo estoy igual, bastante atrasada y no entiendo bien cómo hacer para entregar las prácticas. ¿Alguien sabe cómo entregarlas? Muchas gracias y disculpen las molestias
________________________________
De: Luis Eduardo Guillén Poot ***@***.***>
Enviado: jueves, 20 de octubre de 2022 3:13
CC: Subscribed ***@***.***>
Asunto: Re: [Launch-X-Latam/mision-frontend] Duda sobre el curso. (#26)


Hola. Muchas gracias. 🥺

**Comment by snowingt** [comment]
Hola , las prácticas se suben a tu repositorio, es la forma de entregar . Y tienes tiempo igual, yo empezare este fin de semana a ponerme al dia con todo que he estado vagueando mucho

**Comment by Annamalag** [comment]
hola compañeros me comentan cual es la practica que hay que hacer?

**Comment by GuillenLuis** [comment]
> hola compañeros me comentan cual es la practica que hay que hacer?
> 
> 
> El jue, 20 oct 2022 a la(s) 19:51, snowingt ***@***.***)
> escribió:
> …

Hola. Pues... Cada unidad o tema tiene su propia prática. Al final de cada tema o unidad aparece lo que es la respectiva práctica. En el caso de el tema 1, es hacer la toma de requerimientos para una empresa llamada Abogabot, así como el buyer persona, público objetivo, wireframe y mockup del sistema. Saludos :)

**Comment by lopez-juandavid** [comment]
> Hola , las prácticas se suben a tu repositorio, es la forma de entregar . Y tienes tiempo igual, yo empezare este fin de semana a ponerme al dia con todo que he estado vagueando mucho

Hola explorer. Una pregunta el repositiorio lo creo aparte para subir todo lo relacionado al front end, verdad?.

**Comment by GuillenLuis** [comment]
> Hola , las prácticas se suben a tu repositorio, es la forma de entregar . Y tienes tiempo igual, yo empezare este fin de semana a ponerme al dia con todo que he estado vagueando mucho
> 
> Hola explorer. Una pregunta el repositiorio lo creo aparte para subir todo lo relacionado al front end, verdad?.

Que yo tenga entendido, sí. Tus prácticas las subes a tu repositorio. Justo ahora estaba leyendo la sección General de Teams y era lo que estaban comentando, que nuestras prácticas van subidas a nuestros repos y que esa era la forma de entregar.

**Comment by andres-rita** [comment]
> 

Recuerden que es importante leer todos lo comunicados que han dejado para no tener dudas al respecto, por igual están las grabaciones en Teams.

**Comment by Strato1257** [comment]
te recomiendo primero ver  TODOS los videos y postetrior ir haciendo, para que veas de que se trata, así cuando busques ya estará más alojado en tu mente, por ende ya entederás de a poco a poco.
